### DISCUSSION TITLE
Team features

### DISCUSSION THREAD

**Discussion by LaminasyRetales** [vercel/community#1269] [Help] [answered]
### Summary

Hello, i made a deploy using a team in Vercel.com and i have that premium features until January 20th, my question is, if i dont pay anymore, i will lost my deploy or i just lost my premium features? Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

If you downgrade to Hobby plan (stop paying for Pro Plan), you won't be able to deploy organization repository as well as you can't add new team members. If you have commercial project, it's prohibited to use against free plan: https://vercel.com/docs/concepts/limits/fair-use-policy
